libvmi/convenience.c

```diff
@@ -181,7 +184,7 @@ vmi_convert_str_encoding(
         goto fail;
     }   // if failure
 
-    // conversion success
+// conversion success
```

Comment:
Moving the comment spacing like this looks odd to me. Was astyle complaining about it?